Source/WebKit/UIProcess/API/Cocoa/WKSnapshotConfiguration.mm

```diff
@@ -24,9 +24,13 @@
  */
 
 #import "config.h"
-#import "WKSnapshotConfiguration.h"
+#import "WKSnapshotConfigurationPrivate.h"
 
-@implementation WKSnapshotConfiguration
+@implementation WKSnapshotConfiguration {
+#if !TARGET_OS_IPHONE
```

Comment:
This is an implementation file, so !PLATFORM(IOS_FAMILY) probably?

Reply:
(And below)

Reply:
I'm going to go with `PLATFORM(MAC)` instead.